app/Models/EventUserRegistered.php

```diff
@@ -40,11 +40,11 @@ class EventUserRegistered extends Model
 
     protected $guarded = [];
 
-    public function event(){
+    public function event() {
         return $this->hasOne('App\Models\Event', 'id', 'event_id');
     }
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->